test/sagaHelpers/retry.js

```diff
@@ -0,0 +1,39 @@
+import test from 'tape'
+import lolex from 'lolex'
+import sagaMiddleware from '../../src'
+import { createStore, applyMiddleware } from 'redux'
+import { take, retry } from '../../src/effects'
+
+test('retry fails', assert => {
+  const clock = lolex.install()
```

Comment:
would prefer some simpler test case, not based so heavily on the clock

you can just put a counter variable here, increment with each worker call and just decide in the worker if based on the counter it should fail or succeed, and ofc based on it test retry behaviour

the clock needs to be ticked ofc, but only to move the test-timeline forward

Reply:
I used the same setup as in `throttle` but sure I can try to simplify it.